liferay-lima/liferay-portal
# LPS-169119 Remove usages of toolbar item tag in DM

# What is this about?

https://issues.liferay.com/issues/browse/LPS-169119

Remove usage of legacy tag for toolbar items in DM, using dropdown items instead. As this was the only usage in the whole codebase, remove the tag itself and supporting classes too.

# How do I test it?

- Drop the Documents and Media widget in a content or widget page.
- Configure the portlet, enabling the "Show Actions" checkbox.
- Add a document if necessary.
- Enter the document details by clicking its title in the card.

You should see two main differences with respect to current master:

- The info button looks like an actual info button.
- Instead of a row of button -- one per available actions -- there should be a list of dropdow items to the right of the info button.

The actions should be the same as they were before, and all should work.

# Any pending work?

Yes. Dear frontend developers, I need your help fixing the visuals (see attached screenshot).
![image](https://user-images.githubusercontent.com/729897/203055784-3fea8491-dc43-4d37-baa8-0302d2dd2cf4.png)
